src/ima.js

```diff
@@ -454,7 +454,7 @@ export default class Ima extends BasePlugin {
     this._sdk.settings.setPlayerVersion(this.config.playerVersion);
     this._sdk.settings.setVpaidAllowed(true);
     this._sdk.settings.setVpaidMode(this._sdk.ImaSdkSettings.VpaidMode.ENABLED);
-    if (this.config.setDisableCustomPlaybackForIOS10Plus === 'boolean') {
+    if (typeof this.config.setDisableCustomPlaybackForIOS10Plus === 'boolean') {
```

Comment:
@odedhutzler 
Although this is a good fix, how this related to the adsRenderingSettings?

Reply:
U r right @dan-ziv, added a fix for undefined instead of null on IMA adsRenderingSettings.
Please update your request for changes

Reply:
@odedhutzler I think we can add tests for this - please check